src/test/scala/scalismo/faces/FacesTestSuite.scala

```diff
@@ -28,11 +28,13 @@ class FacesTestSuite extends FunSpec with Matchers {
 
   def randomRGB = RGB(rnd.scalaRandom.nextDouble(), rnd.scalaRandom.nextDouble(), rnd.scalaRandom.nextDouble())
 
+  def randomsRGB: sRGB = sRGB(rnd.scalaRandom.nextDouble(), rnd.scalaRandom.nextDouble(), rnd.scalaRandom.nextDouble())
```

Comment:
After looking at all the code, I think, I would prefer `SRGB` although it looks a bit unfamiliar, given the name of the colorspace "sRGB" - code reading is easier with a capital "S"